Flexible Learning/units_browse_details_submitProcess.php

```diff
@@ -122,6 +126,37 @@
         'attachmentLocation' => $data['evidenceLocation'],
     ]);
 
+    // Invite Parents to Comment
+    if ($roleCategory == 'Student') {
+        $student =  $container->get(StudentGateway::class)->selectActiveStudentByPerson($gibbon->session->get('gibbonSchoolYearID'), $gibbon->session->get('gibbonPersonID'))->fetch();
+        $familyAdults = $container->get(FamilyGateway::class)->selectFamilyAdultsByStudent($gibbon->session->get('gibbonPersonID'))->fetchAll();
+        $familyAdults = array_filter($familyAdults, function ($adult) {
+            return $adult['contactEmail'] == 'Y' && $adult['childDataAccess'] == 'Y';
+        });
+
+        if ($inviteParents == 'Y' && !empty($student) && !empty($familyAdults)) {
```

Comment:
Interesting that you did this with an email rather than a Gibbon notification. I guess email is more reliable for those who might have turned off Gibbon notification emails. No right or wrong, just interesting.

Reply:
I waffled on this one. Notifications are easy to miss, because there's nothing specific in the subject line, and yes, if they're turned off then the recipients wouldn't see this. Perhaps an email, because it can say flexible learning in the subject line, is more likely to be noticed?

Reply:
Email sounds like the right way to go : )